salt/profiles/postgresql.sls

```diff
@@ -43,8 +43,6 @@ profiles-db:
             - profiles-db-user
 
     cmd.run:
-        - owner: {{ pillar.profiles.db.username }}
-        - name: public
```

Comment:
this is weird

Reply:
probably a copy+paste error from the above states. If the intent was to run it as the database user it would have failed and the `name` attribute would have been overwritten in all cases.

Reply:
I think it's clearly a leftover from a `postgres_schema.present` state.